Gemfile

```diff
@@ -7,6 +7,7 @@ group :test do
   gem 'facter', (ENV['FACTER_GEM_VERSION'] || '~> 2.0'),           require: false
   gem 'puppet-lint-classes_and_types_beginning_with_digits-check', require: false
   gem 'puppet-lint-leading_zero-check',                            require: false
+  gem 'puppet-lint-param-types',                                   require: false
```

Comment:
That looks great, however:
https://github.com/opus-codium/puppet-taiga/blob/7d0de5e3011fe879110acb6253c04371d65d5bb4/Gemfile#L1-L2

That look like a good addition to
https://github.com/opus-codium/modulesync_config